src/components/Login/__snapshots__/Login.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Login Should match the snapshot 1`] = `ShallowWrapper {}`;
```

Comment:
Interesting that the snapshot is an empty object.  I dont want that to prevent us from merging and moving forward since its a test (and the functionality is working how we'd like).  I'm going to merge and we can work through the trouble-tests together.